styles/nav.scss

```diff
@@ -121,7 +121,7 @@ $daptiv-subnav-tab-h-pad: 14px;
 
     .daptiv-subnav-tab {
         color: $daptiv-subnav-font-color;
-        display: inline-block;
+        cursor: pointer;
```

Comment:
@boskya This was addressing issue with missing href attributes in link elements. Have added missing hrefs instead.